### DISCUSSION TITLE
Merger quiery

### DISCUSSION THREAD

**Discussion by Adam1990810** [fetchai/uAgents#397] [General]
If someone is holding FET tokens on cold wallets or hot wallets will this automatically be converted asi tokens at a 1:1 ratio?

Or must fet tokens on cold wallets and  hot wallets be put on an exchange for the conversion to asi tokens?

Thank you

**Comment by kshipra-fetch** [comment]
Hey @Adam1990810 

Thanks for your query. @NikoDMT  - could you please take this one on?

**Reply by NikoDMT** [reply]
Yes, funds held on the Fetch.ai mainnet irrespective of the wallet type (cold or hot) will be automatically converted to ASI at the 1:1 ratio once the network upgrade happens. For the time being we've taken the decision to not rush the upgrade however when it happens any FET including staked FET will be converted to ASI.

**Comment by NikoDMT** [comment]
Regardless If you're holding FET cold or hot wallet as long as it's in native form, that is to say in a wallet with a public address beginning with fetch1.. , your FET will be automatically converted to ASI when we proceed with the planned network upgrade. 

If you're holding FET in an erc20 or bep20 form you'll have to manually convert it to ASI through the singularityDAO conversion portal. The portal is essentially a smart contract. The contracts to directly converted FET to ASI for erc20 and bep20 standards aren't live yet, but once they are we'll announce it and post guides on the superintelligence.io website.
